### PR TITLE
fix : v2.0 Issue to avoid replication of data as day changed

### PR DESCRIPTION
**Pull request by PawanPatil19**
fix : fixed v2.0 Issue to avoid replication of data as day changed

### CODE REVIEW

File: src/Watcher/watch_log.py [hunk 1]
```diff
@@ -17,8 +18,9 @@ def get_date():
 
 def update_csv(date, Data):
     user = os.getlogin()
-    filename = "/home/"+os.getlogin()+"/.cache/Watcher/daily_data/"+date+".csv"
+    filename = "/home/"+os.getlogin()+"/.cache/Watcher/raw_data/"+date+".csv"
```
[the remark this answers is not in the document]
Comment (Waishnav) [reply src/Watcher/watch_log.py]: I've changed accordingly you don't need to update.

File: src/Watcher/watch_log.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 import get_windows as x
 import afk as y
 from time_operations import time_difference, time_addition, convert
+import pandas as pd
```
Comment (Waishnav) [inline src/Watcher/watch_log.py:7]: unused module so delete this line

File: src/Watcher/watch_log.py [hunk 3]
```diff
@@ -30,10 +32,10 @@ def update_csv(date, Data):
 # also if usr is AFK then append line
 def import_data(file):
     with open(file, 'r') as f:
```
Comment (Waishnav) [inline src/Watcher/watch_log.py:34]: add error handling here.
FileNotFound error occurs if user installed the watcher and run watcher -s command. 
image

**Review by Waishnav — CHANGES_REQUESTED** [review]

**Comment by PawanPatil19** [comment]
In the install file,  should we change 'raw_data' to 'daily_data'?
`mkdir -p ~/.cache/Watcher/raw_data/`

**Pull request closed without merging**